### PR TITLE
created voice.cfg

### PR DESCRIPTION
**Pull request by cmdflow**

### CODE REVIEW

**Comment by C0kkie** [comment]
You miss a entry in the toc.yml

File: articles/configs/voice.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The `voice.cfg` file is needed if you are running external altv-voice-server. The settings for using it on same machine as altv-server are:
```
Comment (C0kkie) [inline articles/configs/voice.md:1]: You miss the header on the top like

``# The `voice.cfg` configuration file``

**Review by C0kkie — CHANGES_REQUESTED** [review]

**Comment by C0kkie** [comment]
What about a explanation to the config entries like the server.cfg

**Comment by cmdflow** [comment]
Feel free to do so. I don't explain what i don't understand. No idea what was the intention of all those parameters and how they are working internally. Just wanted to help other ppl not to probe against 100 versions of config due to lack of good explanation in docs ;)

**Comment by C0kkie** [comment]
Without a explanation it doesnt make sense to add it

**Comment by cmdflow** [comment]
Well then delete the pull request. so the next one has to ask again if its 127.x or 0.x - i cant explain it. i would if i could

**Comment by DurtyFree** [comment]
@cmdflow thx for the efforts, we understand the lack of documentation here and will try to add some detailed documentation for this soon.

Until then you can follow up in #83 
Closing this for now, thx:)

**Pull request closed without merging**